generator/README.md

```diff
@@ -0,0 +1,23 @@
+# generator
+
+A tool for generating client libraries.
+
+## Example Run
+
+Run the following command from the generator directory:
+
+```bash
+go install github.com/googleapis/google-cloud-rust/generator/cmd/protoc-gen-gclient
+protoc -I cmd/protoc-gen-gclient/testdata/smprotos \
+-I /path/to/googleapis \
+--gclient_out=. \
+--gclient_opt=capture-input=true,language=rust \
+cmd/protoc-gen-gclient/testdata/smprotos/resources.proto \
+cmd/protoc-gen-gclient/testdata/smprotos/service.proto
```

Comment:
I think it may be more readable if we indent these:

```suggestion
    -I /path/to/googleapis \
    --gclient_out=. \
    --gclient_opt=capture-input=true,language=rust \
    cmd/protoc-gen-gclient/testdata/smprotos/resources.proto \
    cmd/protoc-gen-gclient/testdata/smprotos/service.proto
```

Reply:
good idea.